srcpkgs/rnnoise/template

```diff
@@ -0,0 +1,33 @@
+# Template file for 'rnnoise'
+pkgname=rnnoise
+_githash=1cbdbcf1283499bbb2230a6b0f126eb9b236defd
+version=0.4.1
```

Comment:
Where is this version number coming from? We don't usually package software without proper releases, and this just looks like you picked the latest commit...

Reply:
This is the version number coming from the name of the .so. It must be set somewhere in the repo. I'll ask if the maintainer can add a proper release tag.

Reply:
@ericonr IMO we should make an exception out of the "proper releases" rule for this case. It's been about 10 months since @darix asked them to tag releases, but still no response from them.